recipes/colormap/meta.yaml

```diff
@@ -0,0 +1,39 @@
+package:
+  name: colormap
+  version: "1.0.0"
+
+source:
+  fn: colormap-1.0.0.tar.gz
+  url: https://pypi.python.org/packages/dd/de/2ab8312cdf4d4c2bc59b6fef4c51bf616dd431251d858079678c84c6359e/colormap-1.0.0.tar.gz
+  sha256: eb73ca956770ae4e92a071d53daeb3c20d02c8b167797a2784f739db09749edd
+
+build:
+  skip: False
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - setuptools
+    - matplotlib
+    - easydev
+
+test:
+  imports:
+    colormap
+
+about:
+  home: http://packages.python.org/colormap/
+  license: LGPL-3.0
```

Comment:
this requires to change the packages. I would prefer to update the recipes at a later stage if possible.